src/pages/Lend/LoanSearchPage.vue

```diff
@@ -55,5 +48,25 @@ export default {
 		KvMaterialIcon,
 		LoanSearchInterface
 	},
+	data() {
+		return {
+			mdiEarth,
+			mdiFilter
+		};
+	},
 };
 </script>
+
+<style lang="scss" scoped>
+	.better-future {
+		font-size: 1.5rem;
```

Comment:
@mcstover I had to add font styles manually instead of TW classes, for some reason font oriented TW classes are not working

Reply:
Ok let's try clearing the build cache and getting these styles in place using our `tw-` classes.